test/functoria/key_gen.ml.expected

```diff
@@ -0,0 +1,5 @@
+(* Generated by "./test.exe". *)
+
+
+let runtime_keys = List.combine [] []
```

Comment:
Not sure why this is empty...

Reply:
Actually it's empty because the default implementation that I am using do not define any keys :p